src/server/index.js

```diff
@@ -4,18 +4,22 @@ import { MCFAdapter as MCFAdapterClass } from './MCFadapter'
 import { TCGAAdapter as TCGAAdapterClass } from './TCGAadapter'
 import bodyParser from 'body-parser'
 import Pool from 'pg-pool'
+import MongoClient from 'mongodb'
 
 const app = express()
 const port = process.env.PORT || 8080
 const dbURL = new URL(
   process.env.DATABASE_URL || 'postgres://genomedb:genomedb@postgres:5432/genomedb'
 )
+const mongodbURL = process.env.MONGODB_URL
+const mongodbDB = process.env.MONGO_DATABASE
```

Comment:
Naming could be better, how about:
```
const mongoURL = process.env.MONGO_URL
const mongoDatabase = process.env.MONGO_DATABASE
```

Don't forget to update `docker-compose`.